docs/man/futhark-literate.rst

```diff
@@ -194,12 +194,42 @@ The following directives are supported:
   Use ``set term png size width,height`` to change the size to
   ``width`` by ``height`` pixels.
 
-* ``> :audio e``
+* ``> :audio e[; parameters...]``
 
   Creates a sound-file from ``e``, which must be an expression of type ``[]i8``.
   It is interpreted as a 8-bit PCM audio at 44100 Hz and turned into a
   wave-soundfile.
 
+  The optional parameters are lines of the form *key:value*:
+
+  * ``sampling_frequency: <int>``
+
+    The sampling frequency (in Hz) of the input. Defaults to
+    ``44100``.
+
+  * ``codec: <name>``
+
+    The codec of the output. Defaults to ``wav``.
```

Comment:
Which other codecs are likely to work?

Reply:
Depends on ffmpeg. I've just tried flac, mp3, opus, ogg and they all seemed to work. I'll add a note.